### DISCUSSION TITLE
STAR_salmon raw-counts matrix does not have integers

### DISCUSSION THREAD

**Discussion by alessiaartusi** [nextflow-io/nextflow#4838] [Q&A]
Hi everyone,

I have used the nf-core rnaseq pipeline (version 3.14.0) on my RNA-seq fastq data.

I am running nextflow in a SLURM cluster with the following command
```
${nextflow} run nf-core/rnaseq \
	--outdir ${OUT_DIR} \
	-profile slurm,singularity \
	-c ${channel} \
	--input ${samplesheet} \
	--fasta ${GENOME_FASTA} \
	--gtf ${GENOME_GTF} \
	-r 3.14.0 \
	--save_reference
```
The pipeline finished successfully. My objective is to perform a differential expression analysis on the data in output and I wanted to start from raw counts, but the file
```
star_salmon/salmon.merged.gene_counts.tsv
```
which should contain raw counts, is actually a matrix of decimal.
Does anyone know why is that?

Thank you in advance

**Comment by james-bole-pan** [comment]
Same question. I'm just rounding them to integers for now - would like to know if this the correct approach

**Reply by muskang06** [reply]
I am also doing the same and did not find any answer to this question.

**Reply by bentsherman** [reply]
Take this up with the developers of nf-core/rnaseq, they should be able to give an answer
